src/main/java/frc/robot/Constants.java

```diff
@@ -138,6 +138,7 @@ public static class Intake {
   public static class Limelight {
     /** Constants for aiming Limelight. */
     public static class AimingLimelight {
+      /** OUR Network table's name for the limelight */
```

Comment:
Don't put OUR in caps. And also make it so it's saying that it's OUR name for the limelight in networktables. Not networktable's.